Firestore/Source/CMakeLists.txt

```diff
@@ -90,4 +91,83 @@ if(APPLE)
     PUBLIC
     ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public
   )
-endif()
+
+  objc_framework(
+    FirebaseFirestore
+    HEADERS
+      ${FIREBASE_SOURCE_DIR}/Firestore/Source/Public/FirebaseFirestore.h
```

Comment:
This is failing `check.sh`, I tried to use `Public/FirebaseFirestore.h` here but then the imports from integration tests fail